src/device/cuda/wmma.jl

```diff
@@ -565,3 +565,52 @@ wmma_fill_c
         return wmma_fragment{$M, $N, $K, $num_els, $T, wmma_unspecified, wmma_accumulator}($expr)
     end
 end
+
+################################################################################
+# BROADCASTING OVER WMMA_FRAGMENTS
+################################################################################
+
+# Based on broadcasting implementation of Tuples in
+# https://github.com/JuliaLang/julia/blob/master/base/broadcast.jl
+
+
+# Custom broadcast style for wmma_fragments
+struct wmma_fragment_broadcast_style <: Broadcast.BroadcastStyle end
```

Comment:
Just as a sidenote types in Julia are most often camelcase with initial capital letters.

`WMMAFragmentBroadcastStyle`